src/OrbitGl/GraphTrack.cpp

```diff
@@ -89,23 +89,12 @@ void GraphTrack::Draw(GlCanvas* canvas, PickingMode picking_mode, float z_offset
     return;
   }
 
-  // Draw label
-  uint64_t current_mouse_time_ns = time_graph_->GetCurrentMouseTimeNs();
-  auto previous_point = GetPreviousValueAndTime(current_mouse_time_ns);
-
-  double value =
-      previous_point.has_value() ? previous_point.value().second : values_.begin()->second;
-  uint64_t first_time = values_.begin()->first;
-  uint64_t label_time = std::max(current_mouse_time_ns, first_time);
-  float point_x = time_graph_->GetWorldFromTick(label_time);
-  double normalized_value = (value - min_) * inv_value_range_;
-  float point_y = pos_[1] - size_[1] * (1.f - static_cast<float>(normalized_value));
   const Color kBlack(0, 0, 0, 255);
   const Color kWhite(255, 255, 255, 255);
   float text_z = GlCanvas::kZValueEvent + z_offset;
-  DrawLabel(canvas, Vec2(point_x, point_y), std::to_string(value), kBlack, kWhite, text_z);
 
   // Add warning threshold text box and line.
+  Batcher* batcher = canvas->GetBatcher();
```

Comment:
This was here before, but please rename to ui_batcher to be consistent with other tracks.

Reply:
Changed! Thanks!